apps/ledger-live-mobile/src/components/SelectDevice2/DeviceList.tsx

```diff
@@ -0,0 +1,39 @@
+import { Device } from "@ledgerhq/live-common/hw/actions/types";
+import React, { useState } from "react";
+import Item from "./Item";
+import { Button, Flex } from "@ledgerhq/native-ui";
+import { useTranslation } from "react-i18next";
+
+const MAX_LEDGERS_DISPLAYED = 3;
+
+type Props = {
+  deviceList: Device[];
+  handleOnSelect: (device: Device) => void;
+};
+export function DeviceList({ deviceList, handleOnSelect }: Props) {
+  const { t } = useTranslation();
+  const [showAll, setShowAll] = useState(false);
```

Comment:
An alternative when you need a toggle is to use a `useReducer`:
`const [showAll, toggleShowAll] = useReducer(state => !state)`

Reply:
Noted ! Thanks